### DISCUSSION TITLE
Hacktoberfest Data Importer

### DISCUSSION THREAD

**Discussion by diogotcorreia** [firefly-iii/firefly-iii#6478] [General]
Hi,

I'm thinking about contributing to the data importer during this month.

Could you add the `hacktoberfest` topic to the data importer repository as well (like this repository already has)?

Thank you! :rocket:

**Comment by JC5** [comment]
Thanks for reminding me. I'm sorry, but I will not be participating in Hacktoberfest this year. I've removed the tag from the main Firefly III repository. Perhaps next year again!

If you had anything specific you wanted to submit to the Data Importer, let me know!

**Reply by diogotcorreia** [reply]
That's very unfortunate.
May I ask why?

**Reply by JC5** [reply]
Hacktoberfest tends to draw in a lot of newbies that break more than they make.
